crates/metal-mir/src/stmt/import.rs

```diff
@@ -26,7 +23,14 @@ use crate::{parcel::Module, types};
 /// ```
 #[derive(Debug, Clone)]
 pub struct Import {
```

Comment:
Shouldn't this have a visibility field? #

Reply:
imports are just used for tracking metadata, debugging, and linking, in which all three have no reliance on visibility (nor much support too,) that again is a frontend convenience.